main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"html"
+	"log"
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Hello, %q", html.EscapeString(r.URL.Path))
+	})
+
+	log.Fatal(http.ListenAndServe(":8080", nil))
+//Just for fun
```

Comment:
get rid of this comment, it doesn't really mean anything

Reply:
Alright. I just made that because I wanted to understand commit and push better, so was trying to change the file by adding comment and redo the process